tools/quantwiz_iq/quantwiz_iq.xml

```diff
@@ -0,0 +1,215 @@
+<tool id="quantwiz_iq" name="QuantWiz-IQ" version="2.0">
+    <description>Isobaric Quantitation using QuantWiz-IQ</description>
+    <requirements>
+        <requirement type="package" version="2.0">quantwiz-iq</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+cp '$input_file' '$input_file.name'
+&& cat '$new_param_file' > '$iq_params'
+#if $input_purity_file.input_purity_file_option == "history":
+&& cp '$input_purity_file.purity_type' '$input_purity_file.purity_type.name'
+&& echo "Purity_corrections_values=$input_purity_file.purity_type.name" >> '$iq_params'
+#else:
+&& PROG_PATH=`which QuantWiz_IQ`
+&& cp \${PROG_PATH%/*}/Purity_correction/$input_purity_file.purity_type .
+&& echo "Purity_corrections_values=$input_purity_file.purity_type" >> '$iq_params'
+#end if
+
+&& QuantWiz_IQ '$iq_params'
+&& cp '${input_file.name.rsplit('.',1)[0]}_${area_method}_${min_tol}.tsv' '$quant'
+&& cp '${input_file.name.rsplit('.',1)[0]}_${area_method}_${min_tol}_noquant.tsv' '$noquant'
+    ]]>
+    </command>
+    <configfiles>
+        <configfile name="new_param_file"><![CDATA[#slurp
+Spectra_Folder=.
+#if $input_file.ext == "mgf":
+Input_file_extension=mgf
+#else if $input_file.ext == "MGF":
```

Comment:
the extension that you get from Galaxy is always lower-case, so you can remove a few of the if ...

Reply:
Thanks! Done